homeassistant/components/mobile_app/util.py

```diff
@@ -53,3 +57,20 @@ def get_notify_service(hass: HomeAssistant, webhook_id: str) -> str | None:
             return target_service
 
     return None
+
+
+_CLOUD_HOOK_LOCK = asyncio.Lock()
+
+
+@callback
+async def create_cloud_hook(
```

Comment:
Since this function is imported by other modules, I think we should prefix its name with `async_`
```suggestion
async def async_create_cloud_hook(
```